build/mark-new-version.sh

```diff
@@ -105,22 +110,29 @@ gofmt -s -w "${VERSION_FILE}"
 
 echo "+++ Committing version change"
 git add "${VERSION_FILE}"
-git commit -m "Kubernetes version $NEW_VERSION"
+git commit -m "Kubernetes version ${NEW_VERSION}"
 
 echo "+++ Tagging version"
-git tag -a -m "Kubernetes version $NEW_VERSION" "${NEW_VERSION}"
+git tag -a -m "Kubernetes version ${NEW_VERSION}" "${NEW_VERSION}"
+# We have to sleep for a bit so that the timestamp of the beta tag is after the
+# timestamp of the release version, so that future commits are described as
+# beta, and not release versions.
+sleep 5
```

Comment:
Echo what you're doing here, otherwise people may think the script has gone out to lunch.